src/include/doshelpers.h

```diff
@@ -54,6 +54,8 @@
 
 #define DOS_HELPER_MFS_HELPER       0x20
 #define DOS_SUBHELPER_MFS_REDIR_INIT 5
+#define DOS_SUBHELPER_MFS_REDIR_STATE 6
```

Comment:
The service is used in a later commit when lredir2 is changed from performing the init itself. It's a bit of a judgement really whether to add the service in this commit or the later one where it gets used for the first time.